noetic/README.txt

```diff
@@ -0,0 +1,30 @@
+Status
+======
+The project so far is the Roadmap 1. See below. With the caveat that certain libs are unavail. therefore commented out.
+
+Dependencies
+============
+
+eva-owyl --
```

Comment:
Ditch owyl completely. Pretend it no longer exists. There is no way to integrate it with opencog; it's functional purpose is taken over by the AtomSpace.  (Owyl itself was a way of doing "behavior trees" and having a "blackboard" -- the AtomSpace is a far more generic "blackboard" and has all the basics needed for behavior trees and/or a zillion other styles of control.)